change/@azure-msal-common-2020-11-06-12-29-58-mandate-localAccountId.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Mandate localAccount in AccountInfo",
+  "packageName": "@azure/msal-common",
+  "email": "sameera.gajjarapu@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
```suggestion
  "dependentChangeType": "none",
```

Reply:
I swear again, I pushed this change. there is some mess up with beachball changes post script. :(